environments/linux/docker/.dockerignore

```diff
@@ -0,0 +1,2 @@
+.git
+.vscode/*
```

Comment:
Should we add trailing newlines to all files?

Reply:
Fixed in last commit.